OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -1002,9 +1155,17 @@ function Add-TentacleToWorkerPool
             "register-worker",
             "--instance", $name,
             "--server", $octopusServerUrl,
-            "--force"
+            "--force",
+            "--comms-style", $commsStyle
         )
 
+        # Get tentacle version to determine spaces compatiblity
+        if (![String]::IsNullOrEmpty($Space))
```

Comment:
Consider moving this to another function - this code is duplicated in a bunch of spots

Reply:
Created new function and implemented where it fit.